docs/en/Configuration-Settings.md

```diff
@@ -179,6 +197,35 @@ earliest in this list:
 server process including master and worker searches `alluxio-site.properties` in a list paths of
 `${HOME}/.alluxio/`, `/etc/alluxio/` and `${ALLUXIO_HOME}/conf` in order, and will skip the
 remaining paths once this `alluxio-site.properties` file is found.
+4. [Cluster default values](#use-cluster-default). An Alluxio client may initialize its
+configuration based on the configuration fetched from the master nodes as the cluster-wise default.
 
 If no above user-specified configuration is found for a property, Alluxio runtime will fallback to
 its [default property value](Configuration-Properties.html).
+
+To check the value of a specific configuration property and the source of its value, user can use
+the following commandline:
+
+```bash
+$ bin/alluxio getConf --source alluxio.worker.port
+DEFAULT
+```
+
+To list all of the configuration properties with sources:
+
+```bash
+$ bin/alluxio getConf --source
+alluxio.conf.dir=/Users/bob/alluxio/conf (SYSTEM_PROPERTY)
+alluxio.debug=false (DEFAULT)
+...
+```
+
+To list all of the configuration properties by the master with sources (rather than the local
```

Comment:
can we mention that this requires the cluster to be running, while the other getConf commands do not?